tests/utils.go

```diff
@@ -320,6 +339,21 @@ func BeforeTestSuitSetup() {
 	CreatePVC(osWindows, defaultWindowsDiskSize)
 }
 
+func DeployTestingInfrastructure() {
+	// Deploy manifets/testing content
+	// TODO: How to distinguish what manifests are for specific cluster?
```

Comment:
I will have to somehow sort out manifests based on what cluster they are meant for ... do you have any preference how to do that ?

Reply:
something like opensift prefix or suffix is fine with me.